packages/forms/src/styled/checkbox/StyledCheckInput.ts

```diff
@@ -41,7 +41,11 @@ const colorStyles = (props: ThemeProps<DefaultTheme>) => {
   `;
 };
 
-export const StyledCheckInput = styled(StyledRadioInput).attrs({
+export interface IStyledCheckInput {
+  type: string;
+}
```

Comment:
This isn't necessary. You can add `as string` to help with type inference. Like this:

```
type: 'checkbox' as string
```